packages/edit-site/src/components/global-styles/navigation-button.js

```diff
@@ -33,9 +33,7 @@ function NavigationButtonAsItem( props ) {
 }
 
 function NavigationBackButtonAsItem( props ) {
-	return (
-		<NavigatorToParentButton as={ GenericNavigationButton } { ...props } />
-	);
+	return <NavigatorBackButton as={ GenericNavigationButton } { ...props } />;
```

Comment:
The `NavigationBackButtonAsItem` component doesn't seem to be used anywhere in Gutenberg.